README.md

```diff
@@ -280,6 +280,36 @@ func HandleRequest(ctx context.Context, name string) (string, error) {
 }
 ```
 
+**gRPC**
+
+Note: `aws-xray-sdk-go` doesn't currently support streaming gRPC call.
+
+Apply xray gRPC interceptors (`xray.UnaryServerInterceptor` or `xray.UnaryClientInterceptor`) to instrument gRPC unary requests/responses, and the handling code.
+
+**gRPC Client**
+
+```go
+conn, err := grpc.Dial(
+    serverAddr,
+    // use grpc.WithChainUnaryInterceptor instead to apply multiple interceptors
+    grpc.WithUnaryInterceptor(
+        xray.UnaryClientInterceptor(serverAddr),
+    ),
+)
+```
+
+**gRPC Server**
+
+```go
+grpcServer := grpc.NewServer(
+    // use grpc.ChainUnaryInterceptor instead to apply multiple interceptors
+    grpc.UnaryInterceptor(
+        xray.UnaryServerInterceptor(),
+        // or xray.UnaryServerInterceptor(xray.ServerInterceptorWithSegmentNamer(xray.NewFixedSegmentNamer("myApp"))) to use a custom segment namer
```

Comment:
This looks a bit wrong now

Reply:
Do you mean there's a typo or I shouldn't use comments to mention the usage of options?

Reply:
I think typo, it's `xray.UnaryServerINterceptor(xray.NewFixedSegmentNamer(...))` right?

Reply:
By the options pattern and the current implementation, adding a segment namer to the unary server interceptor should look like this:
https://github.com/aws/aws-xray-sdk-go/blob/63d86848b24a520ab8fb049e5e5c0006bbcb7264/xray/grpc_test.go#L262-L265

Reply:
Ah I don't think we need to be redundant with e.g. `ServerInterceptor`. Just `UnaryServerInterceptor(WithContext(ctx), WithSegmentNamer(...))` can be done I think, there's no difference between server / client versions of these options right? If we can simplify these names I think we're good to go